CreeDictionary/CreeDictionary/paradigm/panes.py

```diff
@@ -16,23 +24,58 @@ class ParadigmTemplate:
     def __init__(self, panes: Iterable[Pane]):
         self._panes = tuple(panes)
 
+    @property
+    def max_num_columns(self):
+        """
+        What is the largest amount of columns necessary for this entire paradigm.
+        """
+        return max(pane.num_columns for pane in self.panes())
+
     def panes(self):
```

Comment:
Does this need to be a `yield`? Or could it be turned into a parameter?

I see that this wasn't actually changed in this PR, so feel free to ignore for now.

Reply:
Sorry, definitely meant `property`, not `parameter`.